src/Components/Facility/Consultations/Mews.tsx

```diff
@@ -0,0 +1,192 @@
+import useQuery from "../../../Utils/request/useQuery";
+import routes from "../../../Redux/api";
+
+export const Mews = (props: { consultationId: string }) => {
+  const { data: dailyRoundsData } = useQuery(routes.getDailyReports, {
+    pathParams: {
+      consultationId: props.consultationId,
+    },
+    query: {
+      rounds_type: "NORMAL,VENTILATOR,ICU",
```

Comment:
Only the last "critical care" round needs to be fetched right?

```suggestion
      rounds_type: "VENTILATOR",
```

Reply:
but doing that we will not get the recent log.
What if the patient got back to normal rounds??

Reply:
Oh yea, that makes sense! My bad.